Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -6,7 +6,7 @@ Authors: Patrick Massot, Johannes Hölzl, Yaël Dillies
 import Mathlib.Algebra.CharP.Defs
 import Mathlib.Algebra.Group.Subgroup.Ker
 import Mathlib.Analysis.Normed.Group.Seminorm
-import Mathlib.Topology.Instances.ENNReal.Defs
+import Mathlib.Topology.Instances.ENNReal.Lemmas
```

Comment:
This import increase is in tension with the aim to reduce the imports of this file, see https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Turn.20.60Complex.2Eabs.60.20into.20.60Norm.60
cc @YaelDillies

Reply:
I am interested to know why you need this new import, yes